platform-dom/src/main/java/com/softicar/platform/dom/elements/image/viewer/DomImageViewerRotation.java

```diff
@@ -0,0 +1,56 @@
+package com.softicar.platform.dom.elements.image.viewer;
+
+/**
+ * Enumerates rotation steps (i.e. angles) for {@link DomImageViewer}.
+ *
+ * @author Alexander Schmidt
+ */
+public enum DomImageViewerRotation {
+
+	_0,
+	_90,
+	_180,
+	_270;
+
+	/**
+	 * Returns the next {@link DomImageViewerRotation}.
+	 * <p>
+	 * Wraps around.
+	 *
+	 * @return the next {@link DomImageViewerRotation} (never <i>null</i>)
+	 */
+	public DomImageViewerRotation nextRight() {
+
+		return switch (this) {
+		case _0:
+			yield _90;
+		case _90:
+			yield _180;
+		case _180:
+			yield _270;
+		case _270:
+			yield _0;
+		};
```

Comment:
hmm, interesting new syntax...no need to throw exceptions after the `switch`? nice!

Reply:
yup...